.travis.yml

```diff
@@ -151,6 +166,16 @@ before_install:
   - ulimit -a
   - mkdir builds
   - cd builds
+  # Work in our own virtualenv to isolate from travis-ci packages.
+  - travis_retry pip install -U virtualenv
+  - |
+    if [ "${USE_DEBUG}" == "1"];
+    then
+        virtualenv --python=python3-dbg venv
+    else
+        virtualenv --python=python3 venv
+    fi
+  - source venv/bin/activate
```

Comment:
I've copied the logic from NumPy's debug build, however, once the tests have run, this build does not activate the virtual environment.

Reply:
Travis-CI environment is already a virtualenv.
Nesting venv inside virtualenv doesn't work (bug XXXX in Python issue tracker).

Reply:
@pv there was a missing space before the ] in if ``[ "${USE_DEBUG}" == "1"];`` so it always evaluated to false.

I've managed to configure a local build and it's passing on that now. We'll see if it passes CI once I've made a few other changes.